homeassistant/brands/aeonmatrix.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please remove it; it is only required if two or more integration exists for the same brand.